### ISSUE TITLE
Label tracking meta-issue (edit me to get automatically CC'ed on issues!)

### ISSUE THREAD

**Issue by ezyang** [pytorch/pytorch#24422]
This issue is used by [pytorch-probot](https://github.com/pytorch/pytorch-probot) to manage subscriptions to labels.  To subscribe yourself to a label, add a line `* label - @yourusername`, or add your username to an existing line (space separated) in the body of this issue.

As a courtesy to others, please do not edit the subscriptions of users who are not you.

* module: binaries - @ezyang

**Comment by ezyang** [comment]
We're open for business! Bot should be working now.

**Comment by jerryzh168** [comment]
Does it work for team?

**Comment by ezyang** [comment]
No it doesn't understand slashes. Let me fix that

**Comment by ezyang** [comment]
@jerryzh168 I tried, but I couldn't get teams to work (for some reason, when the bot posts a team mention, it doesn't get linkified). Just list everyone explicitly on this list please.

**Comment by ezyang** [comment]
@gujinghui @PenghuiCheng @XiaobingSuper I tentatively added you to the subscribes for mkldnn. Please feel free to edit the subscriptions and make it more accurate if necessary

**Comment by ezyang** [comment]
@fritzo @neerajprad @alicanb @vishwakftw I tentatively added all of you to the "module: distributions" tag, which means you'll get cc'ed all issues with this label. Please, please feel free to remove yourselves if this is too much spam.

**Comment by cpuhrsch** [comment]
Created module: nestedtensor

**Comment by ezyang** [comment]
@cpuhrsch subscribe yourself to it?

**Comment by ezyang** [comment]
@izdeby I took the liberty of adding you to the CC list for boolean tensor. If you don't want to be cc'ed on these feel free to remove yourself

**Comment by ezyang** [comment]
@raghuramank100 I took the liberty of adding you to the CC list for quantization. If you don't want to be cc'ed on these feel free to remove yourself

**Comment by ezyang** [comment]
@VitalyFedyunin @ngimel @mruberry I tentatively added you to the "performance" label. If you don't want to be cc'ed on these free to remove yourself. I'm also not sure if I've gotten everybody who works on "performance", if there are other relevant people, let's please add them to the label as well.

**Comment by VitalyFedyunin** [comment]
Does it CC when label is applied? When issue/PR submitted? Or both?

**Comment by ezyang** [comment]
Label applied to issue. It doesn't work on PRs yet.

**Comment by ezyang** [comment]
@fmassa I took the liberty of adding you to 'module: vision', feel free to remove yourself if you don't want these issues

**Comment by ezyang** [comment]
@seemethere I added you to 'module: ci' and 'module: binaries'

**Comment by ezyang** [comment]
@malfet would you be interested in getting notifications for "module: build" issues here?

**Comment by malfet** [comment]
> @malfet would you be interested in getting notifications for "module: build" issues here?

@ezyang sure

**Comment by lw** [comment]
The automatic tagging doesn't seem to be working for the `tensorpipe` module: #37784 was tagged with `module: Tensorpipe` but neither I nor @beauby were tagged by the bot. Is it perhaps due to the different capitalization? @ezyang

**Comment by lw** [comment]
@mrshenli renamed the tag and made it lowercase and this fixed the problem, thanks!

**Comment by astaff** [comment]
updated PoCs for windows-related issues (`module: windows`)

**Comment by mrshenli** [comment]
Made the following changes to setup boundary between main triage oncall and subteams' triage oncall:

`jit` -> `oncall: jit`
`module: distributed` -> `oncall: distributed`
`transformer/mha` -> `oncall: transformer/mha`

**Comment by malfet** [comment]
@SplitInfinity added your to `module: onnx`

**Comment by cpuhrsch** [comment]
@zhangguanheng66 - removed you from mha/transformer oncall (happy to change back if you like to stay on).

**Comment by pmeier** [comment]
@asmeurer I've added you to the `module: python array api` as per request in https://github.com/pytorch/pytorch/issues/58743#issuecomment-859377579.

**Comment by pmeier** [comment]
@leofang I've added you to the `module: python array api` as per request in https://github.com/pytorch/pytorch/issues/58743#issuecomment-859570534.

**Comment by pmeier** [comment]
@AnirudhDagar I've added you to the `module: python array api`.

**Comment by pmeier** [comment]
@asi1024 @emcastillo @kmaehashi I've added you to the `module: python array api` as per request in https://github.com/pytorch/pytorch/issues/58743#issuecomment-862042836.

**Comment by albanD** [comment]
@izdeby I removed you from the cc list. You can add yourself back if needed.

**Comment by heitorschueroff** [comment]
Could someone remove my username from the lists here please?

**Comment by ezyang** [comment]
@pmeier did so you're all set

**Comment by awaelchli** [comment]
Note, some labels don't have any subscribers. If a new issue opens with this label, chances are lower that someone will reply. An example is the label `oncall: jit`.

**Comment by drisspg** [comment]
I added myself to: `oncall: transformer/mha `

**Comment by bertmaher** [comment]
Added myself to `module: dynamo` and `module: inductor`

**Comment by ColinPeppler** [comment]
Added myself to `module: inductor`

**Comment by jcaip** [comment]
Added myself to `module: sparse`

**Comment by gokulavasan** [comment]
Added myself and @andrewkho to dataloader and data modules

**Comment by fduwjj** [comment]
@ezyang Do we have a way to remove some emeritus members or they will stay there forever?

**Comment by ekr0** [comment]
Does this work for PRs or or only for Issues ? If it doesn't work for PRs, is there any alternative mechanism to track PRs of interest?

cc @ankitageorge @mhorowitz

**Comment by ezyang** [comment]
We are far less good about labeling PRs, but yes, I think it does work for PRs
